### ISSUE TITLE
Breast inflammation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10092]
**Preferred term label:**
Breast inflammation

**Synonyms**
Mastitis

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C3495439

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012649

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mastitis
Mammary duct ectasia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000060
- label: Mastitis
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: PMID:31791447
- def: Mastitis is defined as an inflammation of the breast and may or may not be accompanied by infection.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10092
- synonyms: Breast inflammation(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to HP:0040407
